packages/v4/patternfly-docs/content/accessibility/card/card.md

```diff
@@ -3,21 +3,109 @@ id: Card
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+
+## Accessibility
+
+To implement an accessible PatternFly **card**:
+
+- Provide an accessible name for the card via the `aria-label` or `aria-labelledby` attribute.
+- Ensure any other PatternFly components within a card follow their own accessibility documentation.
+- Place a set of multiple cards inside of a list container, with a list item wrapped around each card.
+- Give a card list container its own accessible name via the `aria-label` attribute if there is no other surrounding context.
+
+## Testing
+
+At a minimum, a card should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="card-a11y-checkbox-1" label={<span>The card has an accessible name via the <code class="ws-code">aria-label</code> or <code class="ws-code">aria-labelledby</code> attribute.</span>} description="This will differentiate each card when announced to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="card-a11y-checkbox-2" label="Standard keyboard navigation can be used to navigate between focusable elements within a card." description={<span><kbd>Tab</kbd> navigates to the next focusable element, and <kbd>Shift</kbd> + <kbd>Tab</kbd> navigates to the previous focusable element.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="card-a11y-checkbox-3" label="Any PatternFly components within the card follow their own accessibility documentation." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="card-a11y-checkbox-4" label="A set of multiple cards are placed within a list container, with a list item wrapped around each card." description="This can be either an ordered or unordered list. This will allow users to use shortcuts via their assistive technology and will let them know how many cards exist." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="card-a11y-checkbox-5" label={<span>If there's no surrounding context, a card list container has its own accessible name via the <code class="ws-code">aria-label</code> attribute.</span>} description="An example of surrounding context would be a heading element that directly precedes the group of cards." />
+  </ListItem>
+</List>
+
+## React customization
+
+Various React props have been provided for more fine-tuned control over accessibility.
+
+| Prop | Applied to | Reason | 
+|---|---|---|
+| `aria-label` or `aria-labelledby` | `Card` | Adds an accessible name to the card. If the card has a `CardTitle`, the `aria-labelledby` prop should be passed in with a value of the `CardTitle`'s `id` prop. Otherwise the `aria-label` prop should be passed in to act as a title for the card. |
```

Comment:
It may be worth noting here or in the prop description below it that if they use the `div` element for decorative reasons, they should not use an aria-label, since divs should have a role if they use an aria-label.

Reply:
What do you think of, "If you primarily want to use a card for styling or layout purposes, it's recommended to pass in a value of "div" to this prop. When doing so, the `aria-label` or `aria-labelledby` prop should be omitted unless a `role` prop is also passed in."

Possible downside is that we then don't elaborate on what `role` should be passed in which may or may not be an issue I suppose. Could instead mention something along the lines of, "...should be omitted, as these props are not well supported on `div` elements without a `role`".

Reply:
I think i'd opt for the second - it's clearer and gives them a hint about how to fix it further, but in those cases, they likely would need to do more research since they've diverged from the intended case so far.